lib/src/util/windows/wmi.cc

```diff
@@ -49,7 +49,7 @@ namespace facter { namespace util { namespace windows {
         if (FAILED(hres)) {
             throw wmi_exception(format_hresult("failed to create IWbemLocator object", hres));
         }
-        _pLoc = scoped_resource<IWbemLocator *>(move(pLoc),
+        _pLoc = scoped_resource<IWbemLocator *>(pLoc,
```

Comment:
`move` vs `copy` on a scalar primitive (such as a pointer or opaque index) is purely semantic. `move` is generally implemented as a `copy` because that's the simplest operation. There's no real negative to using `move` on a primitive if you don't plan to use it again (otherwise you violate the semantic contract, even though it'll work with all current compilers).

`move` only has a functional difference with objects or arrays (composite value types) that define `move` semantics via a move operator or constructor.